test/test-dists.py

```diff
@@ -23,6 +17,25 @@ def iterDistRows(refSeqs, querySeqs, self=True):
             for ref in refSeqs:
                 yield(ref, query)
 
+
+description = 'Run poppunk sketching/distances'
+parser = argparse.ArgumentParser(description=description,
+                                    prog='poppunk_sketch')
+
+parser.add_argument('--ref-db', help='Prefix of db to test')
+parser.add_argument('--results', help='Prefix of results to compare against')
+
+parser.add_argument('--warn-diff', type=float, default = 0.0002,
+                    help='Absolute difference to WARN for')
+parser.add_argument('--warn-diff-frac', type=float, default = 0.02,
+                    help='Percentage difference to WARN for')
+parser.add_argument('--error-diff', type=float, default = 0.001,
+                    help='Absolute difference to ERROR for')
+parser.add_argument('--error-diff-frac', type=float, default = 0.05,
+                    help='Percentage difference to ERROR for')
+
+args = parser.parse_args()
+
 # Generate distances
 rList = []
 ref = h5py.File(ref_db + ".h5", 'r')
```

Comment:
`ref = h5py.File(ref_db + ".h5", 'r')` should be `ref = h5py.File(args.ref_db + ".h5", 'r')`, according to failed test?

Reply:
Yes, thank you!